tests/foreman/maintain/test_health.py

```diff
@@ -225,7 +222,6 @@ def _finalize():
         sat_maintain.execute('dnf clean all')
 
 
-@pytest.mark.include_capsule
```

Comment:
Do we have separate test running for capsule now ?

Reply:
It does not make sense to run this test for capsule. The disk space checks are not present in capsule. Per Platform devs they are not there on purpose.
https://redhat-internal.slack.com/archives/C04LA44DGKH/p1693567817895669